bitnami/kafka/README.md

```diff
@@ -452,6 +452,7 @@ Note: This option requires creating RBAC rules on clusters where RBAC policies a
 externalAccess.enabled=true
 externalAccess.service.type=LoadBalancer
 externalAccess.service.port=9094
+externalAccess.autoDiscovery.enabled=true
```

Comment:
Auto-discovery should be only enabled for "Option A". This option implies that the user is manually providing the list of external IPs.

Reply:
How do I do that ?